### PR TITLE
Another fix issue with -classpath arguments being ignored on Java 17

### PR DESCRIPTION
**Pull request by Pa4ok**
In [that issue](https://github.com/PaperMC/Paperclip/pull/49) it was said that changing the parent classloader will entail various problems, so here is "directly" solution

### CODE REVIEW

File: java17/src/main/java/io/papermc/paperclip/Paperclip.java [hunk 1]
```diff
@@ -1,21 +1,19 @@
 package io.papermc.paperclip;
 
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
+import java.io.*;
```
Comment (electronicboy) [inline java17/src/main/java/io/papermc/paperclip/Paperclip.java:3]: import on demand is evil

Comment (Pa4ok) [reply java17/src/main/java/io/papermc/paperclip/Paperclip.java:3]: it's an auto imort from idea, maybe someday I'll turn it off...

Comment (yannicklamprecht) [reply java17/src/main/java/io/papermc/paperclip/Paperclip.java:3]: Just configure star imports to only happen when 999 imports for that path are present.

**Comment by DenWav** [comment]
What is the issue here? Also this PR doesn't follow the style of the surrounding code. Re-open a PR if you fix the code style issues and can give more details on the issue this fixes.

**Pull request closed without merging**